### PR TITLE
Add titles to pages

### PR DESCRIPTION
**Pull request by dhruvkb**
## Description

This PR adds titles to all the Openverse pages, which will be passed up to the WordPress site using the message functionality.

## Testing Instructions

0. Check out the branch and run the dev server.
1. Surf around Openverse (try to hit every different type of page) and observe the tab titles.

## Checklist

- [x] My pull request has a descriptive title (not a vague title like `Update index.md`).
- [x] My pull request targets the *default* branch of the repository (`main`) or a parent feature branch.
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added or updated tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no visible errors.

[best_practices]:https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

**Review by zackkrida — APPROVED** [review]
We can iterate on these later to make them more seo-ey, this is an excellent baseline.

**Review by obulat — APPROVED** [review]
Nice addition with attention to details. So many different ways to set the title!

File: src/pages/search/image.vue [hunk 1]
```diff
@@ -9,19 +9,30 @@
...
 import { useLoadMore } from '~/composables/use-load-more'
 
 const ImageSearch = defineComponent({
   name: 'ImageSearch',
   props: propTypes,
   setup(props) {
+    const store = useStore()
+
+    const query = computed(() => store.state.search.query.q)
+    useMeta({ title: `${query.value} - ${this.$t('hero.brand')}` })
```
Comment (obulat) [inline src/pages/search/image.vue:27]: Oops, you can't use `this` in the setup function.

**Review by obulat — CHANGES_REQUESTED** [review]
We can only use `const { i18n } = useContext(); i18n.t('somekey')` when setting the titles inside the `setup` functions.

File: src/pages/search/image.vue (hunk 1 above)
Comment (dhruvkb) [reply src/pages/search/image.vue:27]: Good catch, I'm confused as to why it works locally for me 😖.

Comment (dhruvkb) [reply src/pages/search/image.vue]: Updated in cc7a4f2.

**Review by obulat — APPROVED** [review]

**Pull request merged**